File: primes/js/primes.js
for(i=0;i<50000;i++)
{
	var comp=0;
	for(j=2;j<i;j++)
	{
		var f=i;
		var rf=f/j;
		var ri=Math.trunc(rf);
		if(rf==ri)
		{
			comp=1;
			break;
		}
	}
	if(comp==0)
	{
		console.log(String(i)+" -- simple");
	}
}

File: primes/cpp/primes.cpp
#include <stdio.h>

int main()
{
	for(int i=0;i<50000;i++)
	{
		int comp=0;
		for(int j=2;j<i;j++)
		{
			float f=i;
			float rf=f/j;
			int ri=rf;
			if(rf==ri)
			{
				comp=1;
				break;
			}
		}
		if(comp==0)
		{
			printf("%i -- simple\n",i);
		}
	}
	return 0;
}